### PR TITLE
Add responsible field to Result model

### PR DESCRIPTION
**Pull request by ilcsik**
PL15

### CODE REVIEW

File: EquiTrack/reports/models.py [hunk 1]
```diff
@@ -118,6 +118,7 @@ class Result(MPTTModel):
     geotag = models.ManyToManyField(Location)
     prioritized = models.BooleanField(default=False)
     metadata = JSONField(null=True, blank=True)
+    responsible = models.ForeignKey(UserProfile, null=True, blank=True)
```
Comment (ntrncic) [inline EquiTrack/reports/models.py]: Change foreign key to User model instead of UserProfile. Make sure the migrations sequence is correct.

Comment (ilcsik) [inline EquiTrack/reports/models.py]: Done in https://github.com/unicef/etools/pull/123/commits/a0b95f9000456199aa68b870d5a3bec425393e31

File: EquiTrack/reports/migrations/0021_merge.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```
Comment (robertavram) [inline EquiTrack/reports/migrations/0021_merge.py]: let's try to move away from merges... see if you can manually sequence these based on what's existing on the develop branch

**Comment by ilcsik** [comment]
Do we still need this field? We have reponsible_persons now on ResultWorkplanProperties.

**Comment by robertavram** [comment]
@ilcsik you're correct we have this in the resultworkplanproperties

**Pull request closed without merging**